corehq/apps/zapier/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+from collections import namedtuple
```

Comment:
most of this stuff was just extracted from setup/teardown of a test so I could use it in multiple places